packages/admin-ui/src/common/routes.ts

```diff
@@ -512,6 +512,11 @@ export interface Routes {
    */
   systemInfoGet: () => Promise<SystemInfo>;
 
+  /**
+   * Executes security updates on host
+   */
+  updateSecurity: () => Promise<string>;
```

Comment:
Better name would be `runHostSecurityUpdates`

Reply:
bump

Reply:
fixed on last commit